TrinityCore/TrinityCore
# Core/PacketIO: Implement CMSG_UI_MAP_QUEST_LINES_REQUEST

**Changes proposed:**
Allows the player to see available questlines on the map.

-  Implement CMSG_UI_MAP_QUEST_LINES_REQUEST
-  Load QuestPOIBlob.db2
-  Load  QuestPOIPoint.db2

**Issues addressed:**

Closes #28033

**Tests performed:**

- Does it build
- Tested in-game


**Known issues and TODO list:** (add/remove lines as needed)

- Add _questsOrderByQuestLine in DB2Stores.cpp, to order the quests in the questLine by the order they appear. 
- Add DB2Manager::GetQuestsOrderForQuestLine.
